babel.config.js

```diff
@@ -2,5 +2,12 @@ module.exports = (api) => {
   api.cache(true);
   return {
     presets: ["@babel/preset-env", "@babel/preset-react"],
+    env: {
+      test: {
+        plugins: [
+          "@babel/plugin-transform-runtime"
+        ]
+      }
+    }
```

Comment:
This was added to fix the error: [`regeneratorRuntime` is not defined when running Jest test](https://stackoverflow.com/questions/42535270/regeneratorruntime-is-not-defined-when-running-jest-test)